src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -505,4 +505,44 @@ public function update_order(Request $request)
             $i->save();
         }
     }
+
+    public function relation(Request $request)
```

Comment:
I realize there are a lot that don't exist, but please add doc blocks to any new or modified methods in your PRs.  This will help improve the code moving forward.